composer.json

```
@@ -9,7 +9,7 @@
         "spryker/laminas": "^1.0.0",
         "spryker/propel-orm": "^1.16.0",
         "spryker/queue": "^1.1.2",
-        "spryker/synchronization": "^0.1.0 || ^0.2.0 || ^1.0.0"
+        "spryker/synchronization": "^0.1.0 || ^0.2.0 || ^1.16.0"
```

Self-reminder (to make CI green):
```suggestion
        "spryker/synchronization": "^0.1.0 || ^0.2.0 || latest"
```

---

please don't forget to change